### ISSUE TITLE
ViroFBX link is for osx only?

### ISSUE THREAD

**Issue by trikko** [viromedia/viro#129]
This page:
https://virocore.viromedia.com/docs/3d-objects#section-fbx

> FBX is an expansive and flexible 3D model format supported by most 3D authoring software. To load FBX files, use the ViroFBX script to convert the FBX file into a VRX file. The VRX file can then be loaded using Object3D. The ViroFBX script can be found **here**.

That "here" link is a zip file with a binary that doesn't work on linux. Using "file" command I see:

```
$ file ViroFBX 
ViroFBX: Mach-O 64-bit x86_64 executable, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|WEAK_DEFINES|BINDS_TO_WEAK|PIE|HAS_TLV_DESCRIPTORS>
```

So it seems osx binary. Any link for linux?

**Comment by radvani** [comment]
Hi @trikko 

We currently don't have a Linux binary available. However, adding cross-platform support for the VRX tool is on our backlog and we will keep this issue open until that fix is in. Thanks,

Raj

**Comment by trikko** [comment]
I suggest you to add a note about this on website.

**Comment by radvani** [comment]
Yes, that would be helpful, thanks for the suggestion. We've added a note to the documentation.

**Comment by trikko** [comment]
Isn't this tool open source?

**Comment by SebaSOFT** [comment]
Is this in the list of priorities? How would it not be important for a proprietary file format to have conversion support on open platforms instead of a (again) proprietary one? Not making value judgment, I just want to be able to use the tool in a country where macs are not just expensive as a car, but also unavailable.

**Comment by radvani** [comment]
This is definitely a high priority. We're releasing a mitigation next week by supporting GLTF, which does not require any converter (MacOS or otherwise), so you'll be able to input models that way. The initial GLTF will not have skeletal animation, but we will follow on with another release soon after that includes this as well. After GLTF is finished we're going to loop back to cross-platform FBX. Thanks for your patience!

**Comment by trikko** [comment]
Why don't you publish at least vrx file specs? So we could try to build a converter or exporter in the meanwhile.

**Comment by radvani** [comment]
Sure, it's a *gzipped* protocol buffer (proto3). I've attached the protobuf definition. To create the file, you have to create the protobuf generators for the language you're using using [protoc](https://developers.google.com/protocol-buffers/docs/cpptutorial#compiling-your-protocol-buffers). Then you write out the file, gzip it, and it can be used by Viro.

[Nodes.proto.zip](https://github.com/viromedia/viro/files/2016744/Nodes.proto.zip)

**Comment by trikko** [comment]
File zip is damaged :) Sounds funny!

**Comment by radvani** [comment]
Sorry about that, just re-uploaded it as a .txt file.

**Comment by MarcRibguth** [comment]
Is there any progress in this task? It would be great to have a linux version.

**Comment by dam00n** [comment]
No timeframe on when linux will be supported but we do support GLTF format which has tools and libraries for linux and can convert from FBX -> https://docs.viromedia.com/docs/3d-objects#section-gltf

**Comment by u-can-miracle** [comment]
@radvani could you tell proximate estimate when this convert tool will be available for Linux or Windows?

**Comment by faisal212** [comment]
its been 1.5 years but still window script is not created

**Comment by kbits-Mars** [comment]
The problem has not yet been solved in 2020, there is still no script for windows or linux???

**Comment by asadadams** [comment]
has this been solved yet , still no script for windows?
